PyART/catalogs/core.py

```diff
@@ -87,6 +94,22 @@ def download_simulation(self, ID='BAM_0001', path='.',protocol='https',verbose=F
         # pull with lfs
         os.system('cd {}; git lfs pull'.format(self.core_data_path))
 
+    def cut_at_mrg(self):
+        """
+        Find the global peak of the 22 and cut the waveform at this time.
+        Assuming that this is the merger time. For some wfs with postmerger this
+        might not be true!
+        """
+        # find the peak of the 22 mode
+        _, _, _, _, idx_cut = self.find_max(kind='global', return_idx=True)
+
+        # cut all modes at the same index
+        for key in self.hlm.keys():
```

Comment:
A similar task is achieved by the method `Waveform.cut`, we may want to uniform this